tests/acceptance/cli/ursula/test_federated_ursula.py

```diff
@@ -46,10 +46,10 @@ def test_initialize_ursula_defaults(click_runner, mocker):
     assert result.exit_code == 0
 
     # REST Host
-    assert 'Is this the public-facing IPv4 address' in result.output
+    assert "Is this is the public-facing address of Ursula? " in result.output
```

Comment:
Related change made in `literature.py`
```suggestion
    assert "Is this the public-facing address of Ursula? " in result.output
```

Reply:
Unresolving, the text hasn't been updated...